pkg/collector/container_cgroup_collector.go

```diff
@@ -44,7 +46,7 @@ func (c *Collector) updateCgroupMetrics() {
 		}
 		err := c.ContainersMetrics[key].UpdateCgroupMetrics()
 		// if the cgroup metrics of a container does not exist, it means that the container was deleted
-		if key != c.systemProcessName && err != nil {
+		if key != c.systemProcessName && strings.Contains(err.Error(), "cgroup deleted") {
```

Comment:
How's about cgroups.ErrCgroupDeleted.Error() == err.Error()?
Again, shall we really just ignore the cgroup error if the cgroup metric is enabled? I think we should at least log the error that we ignore.

Reply:
Got it. Could you check why bcc error in the test case? Would it be fixed with rerunning the test.
The rest change looks fine to me.

Reply:
```go
if key != c.systemProcessName && err != nil && strings.Contains(err.Error(), "cgroup deleted") 
```

otherwise this will panic when `err == nil`